docs/sources/k6-studio/set-up/usage-collection.md

```diff
@@ -6,13 +6,14 @@ weight: 300
 
 # Usage collection
 
-By default, k6 Studio sends an anonymous usage report, and crash reports and error data, so that we can track how users are using the product and its features. We use that information to prioritize the roadmap with the features that will benefit a majority of users, and reduce the impact of any changes we make. The data doesn't contain any information about you or what you are testing.
+By default, k6 Studio sends anonymous usage reports, as well as crash reports and error data, so that we can track how users are using the product and its features. We use that information to prioritize the roadmap with the features that will benefit a majority of users, and reduce the impact of any changes we make. The data doesn't contain any information about you or what you are testing.
 
-The contents included in the usage report are:
+A usage report includes the following information:
 
-- The running program's operating system (`mac` or `win`).
+- The event name (for example, `app_installed` or `recording_created`).
```

Comment:
Should I list all events somewhere in this doc?

On the one hand, it's nice to have some transparency, on the other it may make it harder to maintain.

In the next iteration, we want to start including some additional information related to an event (e.g. number of rules of each type in a saved generator), so this would also need to be listed.

Reply:
How many events are there? Could we keep it as a sub-list here? 🤔

Reply:
There are 9 events, but the list will grow. I think I'll add a sub-list for now 👍